domain-models-runtime/src/test/java/org/folio/rest/persist/PostgresClientIT.java

```diff
@@ -643,6 +648,40 @@ public void updateSectionSingleQuote(TestContext context) {
     }));
   }
 
+    @Test
+  public void selectWithTimeoutSuccess(TestContext context) {
+      PostgresClient client = postgresClient();
+      client.getSQLConnection(2000, conn -> {
+        client.selectSingle(conn, "SELECT 1, pg_sleep(1);", context.asyncAssertSuccess());
+      });
```

Comment:
```suggestion
      client.getSQLConnection(2000, asyncAssertTx(context, conn -> {
        client.selectSingle(conn, "SELECT 1, pg_sleep(1);", context.asyncAssertSuccess());
      }));
```
Each async callback needs a context.asyncAssertSuccess or context.asyncAssertFail or asyncAssertTx. Otherwise the test will succeed without executing the SELECT.